### DISCUSSION TITLE
ft232h set_cbus_gpio(pins)

### DISCUSSION THREAD

**Discussion by ghost** [eblot/pyftdi#333] [Q&A]
on my ft232h using this function (set_cbus_gpio(pins)) i get about 8000 gpio switches per second maximum, is this the maximum switching speed limit or can it be faster?

**Comment by ghost** [comment]
> on my ft232h using this function (set_cbus_gpio(pins)) i get about 8000 gpio switches per second maximum, is this the maximum switching speed limit or can it be faster?

я разобрался спасибо

**Comment by eblot** [comment]
I think it is mostly I/O bound by the USB bus and stack. Each cbus update requires a round trip over the whole USB stack.
It is not possible to bufferize and send several cbus commands in a row since these commands rely on control endpoint #0.
 
If you find a way to increase this number, please add a comment, thanks.
